app/helpers/session_timeout_warning_helper.rb

```diff
@@ -1,13 +1,13 @@
 module SessionTimeoutWarningHelper
-  def frequency
+  def session_timeout_frequency
     (AppConfig.env.session_check_frequency || 150).to_i
   end
 
-  def start
+  def session_timeout_start
     (AppConfig.env.session_check_delay || 30).to_i
   end
 
-  def warning
+  def session_timeout_warning
```

Comment:
I rename these method because they're injected into *all* our views, and `start` and `warning` are common names, so I wanted to minimize the chances they'd be shadowed or collide